appcues/src/main/java/com/appcues/util/PackageManagerExt.kt

```diff
@@ -0,0 +1,33 @@
+package com.appcues.util
+
+import android.content.ComponentName
+import android.content.Intent
+import android.content.pm.ActivityInfo
+import android.content.pm.PackageInfo
+import android.content.pm.PackageManager
+import android.content.pm.ResolveInfo
+import android.os.Build
+
+// extensions to handle deprecations in Android 33 of PackageManager functions
+// https://developer.android.com/reference/android/content/pm/PackageManager
+
+internal fun PackageManager.getPackageInfoCompat(packageName: String, flags: Int = 0): PackageInfo =
```

Comment:
concept for these Compat extensions from https://stackoverflow.com/a/74741495

there is a feature request in with Google to support more officially in some form https://issuetracker.google.com/issues/246845196